src/ccmain/control.cpp

```diff
@@ -1368,10 +1370,10 @@ void Tesseract::classify_word_and_language(int pass_n, PAGE_RES_IT *pr_it, WordD
   } else {
     tprintf("no best words!!\n");
   }
-  if (tessedit_timing_debug) {
-    clock_t ocr_t = clock();
-    tprintf("%s (ocr took %.2f sec)\n", word_data->word->best_choice->unichar_string().c_str(),
-            static_cast<double>(ocr_t - start_t) / CLOCKS_PER_SEC);
+  if (timing_debug) {
+    total_time = clock() - total_time;
```

Comment:
Why now to use c++11 `chrono::steady_clock::now()` (https://en.cppreference.com/w/cpp/chrono/steady_clock - *most suitable for measuring intervals*) instead of `std::clock()`? 
AFAIK it should be more platform consistent than `std::clock()'....

Reply:
We can use it any time, but in time critical code we'd have to check that it does not cost more resources than `clock()`.